databases/week2/homework_part4.sql

```diff
@@ -0,0 +1,75 @@
+-- Part 4: Creating a database
+-- Using an entity relationship diagram, design the data model for an application of your choice. This can be anything, previous students have used a small business (with staff, offices, and job titles), a library (with books, genres, racks, members, and a borrowing log), or a farm (with animals, barns, and farmers).
+
+-- Your application must include at least one many-to-many relationship and any supporting tables (linking tables) that are needed. The entity relationship diagram must describe what tables you will need, the columns in these tables, which column is the primary key, and the relationships between tables.
+
+-- Next, using the entity relationship diagram as a starting point, write all the necessary CREATE TABLE statements to create all tables and relationships (foreign key constraints) for this data model.
+
+-- Submit an image or PDF of your entity relationship diagram, and a .sql file with the CREATE TABLE statements.
+CREATE DATABASE customer_service;
+DEFAULT CHARACTER SET = "utf8mb4";
+USE customer_service;
+-- Create table Customer: with columns: id, name, phone, email, adress
+CREATE Table `Customer`(
+    `id` INT UNSIGNED PRIMARY KEY AUTO_INCREMENT,
+    `name` VARCHAR(255) NOT NULL,
+    `phone` VARCHAR(255),
+    `email` VARCHAR(255)
+);
+
+ALTER table `Customer`
+MODIFY COLUMN INT UNSIGNED NOT NULL FOREIGN KEY (`address_id`) REFERENCES `Address`(`id`);
+
+ALTER TABLE `Customer` 
+  ADD CONSTRAINT id_customer_address
+  FOREIGN KEY (`address_id`) 
+  REFERENCES `Address`(`id`);
```

Comment:
suggestion to move in the create statement if possible

Reply:
done